src/Fable.Helpers.ReactNative.fs

```diff
@@ -1443,6 +1477,14 @@ module Props =
 open Props
 module R = Fable.Helpers.React
 
+[<Emit("$0")>]
+// density independent pixels
+let Dip (_: float): ISizeUnit = jsNative
```

Comment:
An alternative to use `Emit` here is dynamic casting which is erased by Fable in runtime: `let Dip (x: float): ISizeUnit = unbox x`